### PR TITLE
[Enhancement](Test) Add test config to recycler

### PR DESCRIPTION
**Pull request by Yukang-Lian**
### What problem does this PR solve?

Issue Number: close #xxx

Related PR: #xxx

Problem Summary:

### Release note

None

### Check List (For Author)

- Test 
    - [ ] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [x] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [x] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [x] No.
    - [ ] Yes. 

- Does this need documentation?
    - [x] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by Yukang-Lian** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by Yukang-Lian** [comment]
run buildall

File: cloud/src/recycler/recycler.cpp [hunk 1]
```diff
@@ -2790,6 +2804,9 @@ int InstanceRecycler::recycle_expired_stage_objects() {
         int64_t expiration_time =
                 duration_cast<seconds>(system_clock::now().time_since_epoch()).count() -
                 config::internal_stage_objects_expire_time_second;
+        if (config::force_immediate_recycle) {
+            expiration_time = INT64_MAX;
```
Comment (gavinchou) [inline cloud/src/recycler/recycler.cpp:2808]: 这个应该是个负数才对?

File: cloud/src/recycler/meta_checker.cpp [hunk 2]
```diff
@@ -173,6 +176,12 @@ bool MetaChecker::check_fdb_by_fe_meta(MYSQL* conn) {
                     MYSQL_ROW row = mysql_fetch_row(result);
                     TabletInfo tablet_info = {0};
                     tablet_info.tablet_id = atoll(row[0]);
+                    if (config::enable_meta_checker_verbose_log) {
```
Comment (gavinchou) [inline cloud/src/recycler/meta_checker.cpp:179]: use VLOG  instead

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

**Review by gavinchou — APPROVED** [review]

**Review by luwei16 — APPROVED** [review]

**Comment by Yukang-Lian** [comment]
run buildall

**Pull request merged**